core/parentchain/light-client/src/light_validation_state.rs

```diff
@@ -17,39 +17,32 @@
 
 //! State of the light-client validation.
 
-use crate::{state::RelayState, Error, RelayId};
+use crate::state::RelayState;
 use codec::{Decode, Encode};
 pub use sp_finality_grandpa::SetId;
 use sp_runtime::traits::Block as ParentchainBlockTrait;
-use std::collections::BTreeMap;
 
 #[derive(Encode, Decode, Clone, Debug)]
 pub struct LightValidationState<Block: ParentchainBlockTrait> {
```

Comment:
For now, I kept the `LightValidationState`, but I am not sure if we want to keep it around in the future.